apps/src/StudioApp.js

```diff
@@ -294,6 +294,7 @@ function showWarnings(config) {
   shareWarnings.checkSharedAppWarnings({
     channelId: config.channel,
     isSignedIn: config.isSignedIn,
+    isOwner: dashboard.project.isOwner(),
```

Comment:
There are a lot of places in this file where we currently do dashboard.project. Might explore changing these all over in a separate PR